products/jbrowse-web/src/rootModel/rootModel.ts

```diff
@@ -279,6 +280,7 @@ export default function RootModel({
                     if (self.pluginsUpdated) {
                       self.reloadPluginManagerCallback(
                         JSON.parse(JSON.stringify(getSnapshot(self.jbrowse))),
+                        JSON.parse(JSON.stringify(sessionSnap)),
```

Comment:
may want to change these to structuredClone, it can break with string >512Mb for stringify+parse (and, may be faster in some cases)

Reply:
Done